packages/storybook-block/src/utils/ensureUrl.ts

```diff
@@ -0,0 +1,10 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export const ensureUrl = (testUrl: string) => {
```

Comment:
As all this does is to check if a url already has a protocol (http or https) you could rename this to `addMissingUrlProtocol`, wdyt?

Reply:
good point - thanks 👍